meta/main.yml

```diff
@@ -8,8 +8,14 @@ galaxy_info:
   platforms:
     - name: EL
       versions:
-        - all
+        - 6
+        - 7
+    - name: Ubuntu
```

Comment:
I would prefer if the additional supported platform would be Debian rather than Ubuntu.